package.json

```diff
@@ -56,9 +59,11 @@
     "webpack-stream": "^3.1.0"
   },
   "build": {
-    "linux": {
-      "synopsis": "Mattermost Desktop"
-    }
+  "app-bundle-id": "org.mattermost.desktop",
```

Comment:
Please keep indent.

@it33 Mattermost has two kind of domains. Which is better, `.org` or `.com`?